File: intersection_two_arry_27.pl
#!/usr/bin/perl

### Intersection of two Array ###
print("Enter a Stirng separated by space: ");
$str1=<STDIN>;
chomp ($str1);
@arry=split (/\s+/,$str1);

print("Enter a Stirng separated by space: ");
$str2=<STDIN>;
chomp ($str2);
@arry1=split (/\s+/,$str2);
@result;
foreach $word (@arry){
	foreach $sec (@arry1){
	#print("$word $sec\n");
		if($word eq $sec){
			push @result, $word;
			#print("@result\n");
			last;
		}
	}
}
print("@result\n");

File: PYTHON/intersection_two_arry_27.py
#!/usr/bin/env python3
#######################################################
# Author: Arun Kumar
# Date: 06-Feb-2026
# Filename: intersection_two_arry_27.py
# Description: Find the intersection of two arrays/lists
#######################################################

# Intersection of two Arrays
str1 = input("Enter a String separated by space: ")
arry = str1.split()

str2 = input("Enter a String separated by space: ")
arry1 = str2.split()

result = []
for word in arry:
	for sec in arry1:
		if word == sec:
			result.append(word)
			break

print(result)
